### ISSUE TITLE
Firmware cannot flash on led-controller board.

### ISSUE THREAD

**Issue by xoseperez** [xoseperez/espurna#133]
Originally reported by: **Sascha Bejer (Bitbucket: [sascha_bejer](https://bitbucket.org/sascha_bejer), GitHub: Unknown)**

----------------------------------------

I have 2 of the LED Controllers from your Blog.
The firmware flashes fine with platformIO, esptool and esptool.py

However, it does not start with boot log "cksum 0xef csum 0xef csum error"
Flash ID is 0xe5 0x4014 a [PN25F08B](www.xtxtech.com/upfile/2016082517095182.pdf).
Same result with a Blink example from arduino ide.

Now, this seems to be more of a general issue but since you have flashed the same board can you check if yours is somehow different in flash type? (The SoM is not silkscreen ESP-12F and has no shielding can on mine.)

Bottom line, i have 2 borked boards and a firmware backup i cannot apply as well...
